nixos/modules/services/web-apps/gancio.nix

```diff
@@ -0,0 +1,273 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.gancio;
+  isLocalHostName =
+    cfg.hostName == "localhost"
+    || (lib.hasSuffix ".local" cfg.hostName)
+    || (lib.hasSuffix ".localhost" cfg.hostName);
+  json = pkgs.formats.json { };
+in
+{
+  options.services.gancio = {
+    enable = lib.mkEnableOption (
+      lib.mdDoc ''
+        A shared agenda for local communities.
+        The first user to register, using `admin` as email, will be the initial gancio instance administrator.
```

Comment:
```suggestion
        The first user to register, using `admin` as email, will be the initial gancio instance administrator
```

Reply:
removing that last part as the last versions of gancio create the admin user during the install process.